config/deploy/staging.hcl

```diff
@@ -15,9 +15,14 @@ job "dpulc-staging" {
     auto_revert       = true
   }
   group "web" {
-    count = 1
+    count = 2
     network {
       port "http" { to = 4000 }
+      port "epmd" { static = 6789 }
+      # Add the consul DNS loopback, so we can use consul queries.
+      dns {
+        servers = ["10.88.0.1", "128.112.129.209", "8.8.8.8", "8.8.4.4"]
```

Comment:
Where did we get these?

Reply:
The first one is the loopback address for podman, the second was DNS from the host, which I assume is the org's DNS server - I'm not sure if it's important now 🤔